datafusion/core/tests/sqllogictests/test_files/timestamps.slt

```diff
@@ -261,13 +261,19 @@ SELECT INTERVAL '8' MONTH + '2000-01-01T00:00:00'::timestamp;
 ----
 2000-09-01T00:00:00
 
-# Interval columns are created with timestamp subtraction in subquery since they are not supported yet
 statement ok
-create table foo (val int, ts1 timestamp, ts2 timestamp) as values 
-(1, '2023-03-15T15:00:20.000000123'::timestamp, '2023-01-20T23:00:00.000000099'::timestamp),
-(2, '2023-02-28T12:01:55.000123456'::timestamp, '2000-02-23T11:00:00.123000001'::timestamp),
-(3, '2033-11-02T23:22:13.000123456'::timestamp, '1990-03-01T00:00:00.333000001'::timestamp),
-(4, '2003-07-11T01:31:15.000123456'::timestamp, '2045-04-11T15:00:00.000000001'::timestamp);
+create table foo (val int, ts1 timestamp, ts2 timestamp, i interval) as values
+(1, '2023-03-15T15:00:20.000000123'::timestamp, '2023-01-20T23:00:00.000000099'::timestamp, '1 day'::interval),
```

Comment:
I just picked these intervals randomly -- I would be happy to change them if anyone has better suggestions